x-pack/plugins/alerting/server/lib/alert_summary_from_event_log.ts

```diff
@@ -106,6 +87,37 @@ export function alertSummaryFromEventLog(params: AlertSummaryFromEventLogParams)
     }
   }
 
+  for (const event of executionEvents.reverse()) {
+    const timeStamp = event?.['@timestamp'];
+    if (timeStamp === undefined) continue;
+    const action = event?.event?.action;
+
+    if (action === undefined) continue;
+    if (action !== EVENT_LOG_ACTIONS.execute) {
```

Comment:
Is this check necessary? Aren't we filtering for only this type from the query?

Reply:
This check makes it easier to run tests against this function so that we don't have to filter non-execution events from the `eventFactory`.